lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -791,6 +797,49 @@ class LineChartPainter extends AxisChartPainter<LineChartData> with TouchHandler
     }
   }
 
+ void drawRangeAnnotation(Canvas canvas, Size viewSize) {
+    if (data.rangeAnnotations == null) {
+      print('is null');
```

Comment:
This print is unnecessary and verbose, please remove the print line.

Reply:
6fce476